cleiton-ferreira/1.ii/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
The content of this page is in English or in Portuguese?? How about change it to `lang="pt-br"`